mantidproject/mantid
# SliceViewer - Fix squished 2D Workspace plot when transposing the data

**Description of work.**
This PR fixes a bug which caused the transpose of a matrix workspace on the SliceViewer to be squished.

The problem was caused by using the wrong `extents` to interpolate y data to the count data when the `transpose` kwarg was set to `True`.

**To test:**
1. Load a 2D workspace such as MAR1160 (training data)
2. Open the SliceViewer
3. Transpose the image by using the `x` and `y` buttons.
4. The data should be transposed ok. The colourbar should also stay the same colour intensity.

Fixes #29487

This is a regression so no release notes are required.
